jujugui/static/gui/src/app/components/header-breadcrumb/header-breadcrumb.js

```diff
@@ -65,15 +65,22 @@ YUI.add('header-breadcrumb', function() {
 
     render: function() {
       var auth = this.props.authDetails;
-      var loading = auth && auth.loading;
-      var username = loading ? '...' : auth && auth.user || 'anonymous';
-      return (
-        <ul className="header-breadcrumb">
+      var userItem;
+      if (auth && (auth.user || auth.loading)) {
```

Comment:
I might be missing something here but it seems strange that you sometimes might have a different breadcrumb structure here. It might be more helpful to have the anonymous user there so you can go back up to your profile and log in to the charmstore and see your bundles etc.

Not sure if there's a conversation I missed out on that makes sense of this...